### ISSUE TITLE
Hide unused "Promotion options" node fields

### ISSUE THREAD

**Issue by TravisCarden** [phenaproxima/starshot-prototype#130]
Every node edit form in Drupal has a "Promotion options" fieldset containing "Promoted to front page" and "Sticky at top of lists" fields, but they don't automatically do anything. And they don't have help text explaining what effect they _do_ have even when they _are_ used. We're using them on "Blog post" but not elsewhere. We should hide them any time we're not using them.

**Comment by phenaproxima** [comment]
I'm okay with hiding these from the Page and Event content types for now, since they are, as you point out, not used.

**Comment by boulaffasae** [comment]
"Promoted to front page" and "Sticky at top of lists" fields are actually utilized in our Frontpage view configuration (/admin/structure/views/view/frontpage). Specifically:

- Filter criteria: We have "Content: Promoted to front page (= Yes)" set, which means only content marked as "Promoted to front page" will be displayed.

- Sort criteria: We use "Content: Sticky at top of lists (desc)," so content marked as "Sticky at top of lists" will appear before other content.

Given this configuration, I would advise against hiding these options. Removing them would require us to reconfigure the front page view to adjust or remove both the filter and sort criteria. These fields are potentially useful, and we might want to keep them available for future use.

**Comment by phenaproxima** [comment]
I agree with you, @boulaffasae. I sort of doubt that these fields are causing enough clutter to be worth hiding.

**Comment by TravisCarden** [comment]
I'm not sure I was fully understood. Let me try to be really unambiguous: I'm only talking about hiding the fields on the node edit screen of content types where they're not used. I'm not talking about disabling the feature so it can never be used anywhere or changing how views are configured. I'm not talking about hiding the fields everywhere they appear. The problem I'm highlighting is that there are user-facing forms where those fields appear but don't do anything, leading to a confusing experience for content authors and site builders. It doesn't make sense to promote a "Basic page" to the front page or make it sticky at the top of lists. Hence I think we should hide them _on that specific form display_.

**Comment by boulaffasae** [comment]
Hi Travis, yes sure it should be okey to hide them from specific content types that we think no one will be using them.
